.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  push:
+    paths:
+      - '.github/workflows/test.yml'
+      - 'src/*'
+      - 'profiling/*'
+      - 'benchmarks/*'
+
+jobs:
+  test_cpython:
+    name: CPython ${{ matrix.py }} for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
```

Comment:
Would Windows be supported as well?

Reply:
should be, only hang up is the rustup setup. It's blowing up on my use of wget which is a relic of an earlier attempt.